### PR TITLE
Add SPP Number to inventory list view

### PR DESCRIPTION
**Pull request by agalbrai**
image

### CODE REVIEW

File: frontend/src/features/properties/list/PropertyListView.test.tsx [hunk 1]
```diff
@@ -197,7 +197,6 @@ describe('Property list view', () => {
           netBook: 0.0,
           assessedLand: 958000.0,
           assessedLandDate: '2018-01-01T00:00:00',
-          appraised: 0.0,
```
Comment (agalbrai) [inline frontend/src/features/properties/list/PropertyListView.test.tsx:200]: this field must have been removed from the type

File: frontend/src/features/properties/list/columns.tsx [hunk 2]
```diff
@@ -33,6 +41,19 @@ const spacing = {
   xxlarge: unit * 8,
 };
 
+const getProjectLinkNoDrafts = (namespace: string = 'properties') => (cellInfo: any) => {
+  const projectNumbers = _.filter(cellInfo.value, (p: string) => !p.includes('DRAFT'));
```
Comment (agalbrai) [inline frontend/src/features/properties/list/columns.tsx:45]: don't include draft projects

Comment (Fosol) [reply frontend/src/features/properties/list/columns.tsx:45]: Is there a reason not to include the draft projects?

Comment (agalbrai) [reply frontend/src/features/properties/list/columns.tsx:45]: The story says "only show for approved surplus projects" which I assumed meant no drafts?

Comment (Fosol) [reply frontend/src/features/properties/list/columns.tsx:45]: hmmmm, I doubt that is correct, but Mikaela did put that in as a requirement.  We can get a new story if she wants it different.

**Review by Fosol — APPROVED** [review]

**Review by jdtoombs — APPROVED** [review]

**Comment by devinleighsmith** [comment]
The screenshot just shows the one project number, does this look ok when there is multiple?

**Review by devinleighsmith — APPROVED** [review]

**Comment by agalbrai** [comment]
> The screenshot just shows the one project number, does this look ok when there is multiple?

image
seems to work just fine :)

**Pull request merged**